spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -124,30 +131,107 @@ public PubSubTemplate pubSubTemplate(PublisherFactory publisherFactory,
 				.setMessageConverter(pubSubMessageConverter);
 	}
 
+	@Bean
+	@ConditionalOnMissingBean(name = "subscriberSystemExecutorProvider")
+	public ExecutorProvider subscriberSystemExecutorProvider() {
+		return null;
```

Comment:
Let's not provide `null` beans. If we don't have any opinion, let's not provide any beans at all.

Reply:
I didn't know these didn't need to be specified if they were nullable! Will remove.